src/Shell/Process.php

```diff
@@ -249,20 +254,94 @@ public function onSuccess(Closure $closure = null)
         return $this;
     }
 
+    /**
+     * @return mixed
+     */
+    public function getStdin()
+    {
+        return $this->pipes[static::STDIN];
```

Comment:
What's the use case for exposing the internal state?

Reply:
I created a [test](https://github.com/nickel715/shell-command/commit/a417c0c8a50a616efa531862c0f745b834233c02#diff-0e68ccb1950c89e6333e31c603cbe089R112) for the use case